ipfx/qc_feature_extractor.py

```diff
@@ -0,0 +1,280 @@
+from . import stim_features as stf
+from . import epochs as ep
+from . import error as er
+import logging
+import numpy as np
+from . import qc_features as qcf
+
+
+def cell_qc_features(data_set, manual_values=None):
+    """
+
+    Parameters
+    ----------
+    data_set : MiesDataSet
+        dataset
+    manual_values : dict
+        default (manual) values that can be passed in through input.json.
+
+
+    Returns
+    -------
+    output_data : dict
+        cell qc features
+    tag_list : list
+        tag list
+
+    """
+    if manual_values is None:
+        manual_values = {}
+
+    output_data = {}
+    tag_list = []
+    ontology = data_set.ontology
+    # measure blowout voltage
+    try:
```

Comment:
I wonder if it's worthwhile to separate each of these `try` and `except` blocks into their own functions? Your comments seems to suggest that there could be a `measure_blowout_voltage`, `measure_electrode_zero`, `measure_clamp_seal`, `measure_input_and_series_resistance` functions.

Reply:
I have refactored this block of code  so that each feature is evaluated by the corresponding function. Thanks.